packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -212,7 +212,9 @@ function WorklistDataGrid(props) {
   const [focusedCol, setFocusedCol] = useState(0);
   const [ariaLiveMessage, setAriaLiveMessage] = useState(null);
   const [cellAriaLiveMessage, setCellAriaLiveMessage] = useState(null);
-  const rowSelection = useRef({ isRowSelectionModeToggledByGrid: false });
+  const isRowSelectionModeToggledByGrid = useRef(false);
+  const inShiftUpDownMode = useRef(false);
+  const multiSelectRange = useRef({ start: null, end: null });
```

Comment:
If the start index and end index are numbers, should the unset value be -1 instead of null?

Reply:
Yes, they are indexes. The null makes it clear that the value is not defined.